### PR TITLE
Fix undefined in team invitation email

### LINKED ISSUE #5641: Invite email text undefined: ... has started undefined for your team ...
## How to reproduce
From the team page send an invite link via email. The team has 3 retros and 1 poker meeting open if that's relevant.

## Expected behaviour
Invite email looks nice and has all values defined

## Actual
Invite email contains text:
> Hi georg+testinvite@example.com,
>
> Georg Bremer (georg@example.com) has started undefined for your team (Georgs test team). Just a few clicks and you’re in!

### Acceptance Criteria (optional)
* [ ] Invite email does not contain undefined text

**Estimated effort:** 1 hour to triage. More if root cause is already identified.

### PR DESCRIPTION
**Pull request by igorlesnenko**
Resolves #5641

The problem existed only for poker meeting.
Also, fixed Icebreaker->Tips->Lean more link for poker meeting

image

**How to test:**

* Create a new team
* Create a poker meeting
* Send an email invite
* See no `undefined` in email text
* Go to icebreaker step of a poker meeting on mobile view
* Click on Tips at the bottom menu
* Click Lear more
* See the link opened correctly

### CODE REVIEW

File: packages/client/components/MeetingHelp/CheckInHelpMenu.tsx
Comment (Dschoordsch) [inline packages/client/components/MeetingHelp/CheckInHelpMenu.tsx:14]: +2 Nice find!

**Review by Dschoordsch — APPROVED** [review]
Tested, works.

**Comment by Dschoordsch** [comment]
@igorlesnenko  From my point of view this is small enough to get merged without maintainer review. Maybe just get a second pair of eyes on it.

**Review by BartoszJarocki — APPROVED** [review]
great find! I agree with @Dschoordsch, should be safe to merge 👍🏻

**Pull request merged**
